packages/yoga/src/Result/native/Result/TextWithBadge/index.tsx

```diff
@@ -11,47 +10,10 @@ interface TextWithBadgeProps {
   title: string;
 }
 
-const SCREEN_PADDINGS = 20;
-const CONTENT_MARGINS = 20;
-const AVATAR_CONTENT_MARGINS = 16;
-const BADGE_LIMIT = 20;
-
-const TextWithBadge = ({
-  avatarWidth,
```

Comment:
Can you remove avatarWidth from TextWithBadgeProps interface? It looks like it is no longer being used